extension/jaegerremotesampling/options.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package jaegerremotesampling
+
+import (
+	"time"
+)
+
+// Options holds configuration for the static sampling strategy store.
+type Options struct {
+	// StrategiesFile is the path for the sampling strategies file in JSON format
+	StrategiesFile string
+	// ReloadInterval is the time interval to check and reload sampling strategies file
+	ReloadInterval time.Duration
+	// Flag for enabling possibly breaking change which includes default operations level
+	// strategies when calculating Ratelimiting type service level strategy
+	// more information https://github.com/jaegertracing/jaeger/issues/5270
+	IncludeDefaultOpStrategies bool
```

Comment:
this does not seem to be populated or used

Reply:
It is used by the filesource(currently in newProvider in provider.go, will update the PR soon):

```go
	if !h.options.IncludeDefaultOpStrategies {
		h.logger.Warn("Default operations level strategies will not be included for Ratelimiting service strategies." +
			"This behavior will be changed in future releases. " +
			"Cf. https://github.com/jaegertracing/jaeger/issues/5270")
		h.parseStrategies_deprecated(strategies)
	} else {
		h.parseStrategies(strategies)
	}
```